netci.groovy

```diff
@@ -15,11 +15,16 @@ def static getBuildJobName(def configuration, def os) {
 }
 
 [true, false].each { isPullRequest ->
-    ['Debug', 'Release'].each { configuration ->
+    ['Debug', 'ci_part1', 'ci_part2'].each { configuration ->
```

Comment:
@enricosada 

I think we can do a bit more test and build refactoring, however, right now I am aiming to get the tests passing without requiring being an Administrator.  Once we have that we can see what we can do with test configurations.

When I started I thought we would have a Debug build configuration, to ensure that we could build and run the stuff under the debugger, and the ci_part1 and ci_part2 to continue to ensure that all of the tests run correctly.  Since both of these do a complete release build it did not seem necessary to have a separate build Release validation only step.

I am however thinking that we can probably split coreclr build and test out of ci_part2 and move it to a ci_part3, that way we should get a bit more parallelism out of the game.